Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -226,7 +237,7 @@
             "type": 2,
             "description": "Choose a time grain or property to create x-axis for users.",
             "isRequired": true,
-            "query": "let grainList = datatable(value:string, label:string, seconds:string, group: string, selected: string)[\r\n'3m', '3 minutes', '3600', 'Time Range', 'true',\r\n\t'1h', '1 hour', '86400', 'Time Range', 'true',\r\n\t'2h', '2 hours', '172800', 'Time Range', 'true',\r\n\t'3h', '3 hours', '259200', 'Time Range', 'true',\r\n\t'1d', '1 day', '604800', 'Time Range', 'true',\r\n\t'12h', '12 hours', '604800', 'Time Range', '',\r\n\t'1d', '1 day', '1209600', 'Time Range', 'true',\r\n\t'1d', '1 day', '2592000', 'Time Range', 'true',\r\n\t'7d', '7 days', '2592000', 'Time Range', '',\r\n\t'1d', '1 day', '5184000', 'Time Range', 'true',\r\n\t'7d', '7 days', '5184000', 'Time Range', '',\r\n\t'1d', '1 day', '7776000', 'Time Range', 'true',\r\n\t'7d', '7 days', '7776000', 'Time Range', ''\r\n\t];\r\n let timeTable = grainList\r\n | where seconds == '{TimeRange:seconds}'\r\n | project value, label, group, selected;\r\nlet customDimension = datatable(value:string, label:string, group: string){formattedCustomDimensions};let dimensionList = datatable(value:string, label:string, group: string) {usageStandardDimension};\r\ntimeTable\r\n | union //showCohort,\r\n dimensionList, customDimension\r\n | project value, label, group, selected",
+            "query": "let grainList = datatable(value:string, label:string, seconds:string, group: string, selected: string)[\r\n'3m', '3 minutes', '3600', 'Time Range', 'true',\r\n\t'1h', '1 hour', '86400', 'Time Range', 'true',\r\n\t'2h', '2 hours', '172800', 'Time Range', 'true',\r\n\t'3h', '3 hours', '259200', 'Time Range', 'true',\r\n\t'1d', '1 day', '604800', 'Time Range', 'true',\r\n\t'12h', '12 hours', '604800', 'Time Range', '',\r\n\t'1d', '1 day', '1209600', 'Time Range', 'true',\r\n\t'1d', '1 day', '2592000', 'Time Range', 'true',\r\n\t'7d', '7 days', '2592000', 'Time Range', '',\r\n\t'1d', '1 day', '5184000', 'Time Range', 'true',\r\n\t'7d', '7 days', '5184000', 'Time Range', '',\r\n\t'1d', '1 day', '7776000', 'Time Range', 'true',\r\n\t'7d', '7 days', '7776000', 'Time Range', ''\r\n\t];\r\n let timeTable = grainList\r\n | where seconds == {standardSeconds}\r\n | project value, label, group, selected;\r\nlet customDimension = datatable(value:string, label:string, group: string){formattedCustomDimensions};\r\nlet dimensionList = datatable(value:string, label:string, group: string) {usageStandardDimension};\r\ntimeTable\r\n | union //showCohort,\r\n dimensionList, customDimension\r\n | project value, label, group, selected",
```

Comment:
why is all this doing day/timerange stuff with queries and math instead of using timerange params and standard things that way?

Reply:
because grain in "By value" dropdwon needs a mutiple map like this 
![image](https://user-images.githubusercontent.com/48222200/84464232-cd39ad80-ac28-11ea-9a4f-8d96c4b5d0d3.png)